ee/debug/checkups/init_logs_darwin.go

```diff
@@ -0,0 +1,37 @@
+package checkups
+
+import (
+	"archive/zip"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+func writeInitLogs(_ context.Context, logZip *zip.Writer) error {
+	stdMatches, err := filepath.Glob("/var/log/kolide-k2/*")
+	if err != nil {
+		return fmt.Errorf("globbing /var/log/kolide-k2/*: %w", err)
+	}
+
+	var lastErr error
+	for _, f := range stdMatches {
+		out, err := logZip.Create(filepath.Base(f))
```

Comment:
I think you could use `addFileToZip` if you want?

Reply:
I tried that, I don't like the way it creates each dir along the way, but I don't feel too strongly if other perfer it

```
./Init Logs/
├── init_logs.zip
├── summary.log
└── var
    └── log
        └── kolide-k2
            ├── launcher-stderr.log
            ├── launcher-stderr.log.flaremeta
            ├── launcher-stdout.log
            └── launcher-stdout.log.flaremeta
```

Reply:
Nah it's fine as-is, then!